task/runner.go

```diff
@@ -391,9 +410,16 @@ func humanizeError(err error) error {
 	if err == nil {
 		return nil
 	}
-
 	errMsg := strings.ToLower(err.Error())
 
+	var catErr CatalystError
+	if errors.As(err, &catErr) {
+		if strings.Contains(errMsg, "unsupported input pixel format") {
+			return errors.New("unsupported input pixel format, must be 'yuv420p' or 'yuvj420p'")
```

Comment:
I think the supported formats are defined into `supportedPixelFormats`, maybe we can log them here so when we increase our support we have a single point to edit

Reply:
It's a different config of supported stuff, and we don't read into that static list on our side when using Catalyst. So I just hardcoded them here while we don't get a better error from Catalyst like including the supported list or the file format